x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts

```diff
@@ -73,7 +74,8 @@ export class InfraLogEntriesDomain {
   public async getLogEntriesAround(
     requestContext: RequestHandlerContext,
     sourceId: string,
-    params: LogEntriesAroundParams
+    params: LogEntriesAroundParams,
+    customColumns?: LogEntriesRequest['columns']
```

Comment:
Just a thought: Is `customColumns` on the server side still an appropriate term? Would `columnOverrides` be better?

Reply:
Yeah I like `columnOverrides` more. I'll update it.